src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,70 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static ManufacturerService manufacturerService;
+    private static DriverService driverService;
+    private static CarService carService;
+
     public static void main(String[] args) {
-        // test your code here
+        manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        carService = (CarService) injector.getInstance(CarService.class);
+        driverService = (DriverService)injector.getInstance(DriverService.class);
+        init();
+        carService.getAll().forEach(System.out::println);
+        carService.getAllByDriver(1L).forEach(System.out::println);
```

Comment:
fix it everywhere in your project (with drivers and manufacturers too)
```suggestion
        List<Car> cars = carService.getAll();
        cars.forEach(System.out::println);
        carService.getAllByDriver(cars.get(0).getId()).forEach(System.out::println);
```